os/developer-guides.md

```diff
@@ -0,0 +1,20 @@
+# Developer Guides
+
+Most users will never have to build Container Linux from source or modify it in any way. If you have a need to modify Container Linux, we provide an SDK that allows you to build your own developer images. We also provide OEM functionality for cloud providers and other companies that must customize Container Linux to run within their environment.
+
+* [Modifying Container Linux][mod-cl]
+* [Building development images][dev-images]
+* [Building production images][production-images]
+* [Building custom kernel modules][kernel-modules]
+* [SDK tips and tricks][sdk-tips]
+* [Disk layout][disk-layout]
+* [Kola integration testing framework][mantle-utils]
+
+
+[sdk-tips]: sdk-tips-and-tricks.md
+[disk-layout]: sdk-disk-partitions.md
+[production-images]: sdk-building-production-images.md
+[mod-cl]: sdk-modifying-coreos.md
+[dev-images]: sdk-building-development-images.md
+[kernel-modules]: kernel-modules.md
+[mantle-utils]: https://github.com/coreos/mantle/#kola/
```

Comment:
The trailing slash here breaks the anchor link

Reply:
oof. I was told trailing slash required for repo links - due to the doc sync process. Maybe we should link to README.md#kola instead?

Reply:
I'm surprised that's necessary here, especially given that this url wont' be rewritten.

Linking to README.md would be okay though.